Preconfigured env files per network

piersy opened the pull request:
Instead of providing instructions on how to edit a template env file
we now provide a ready to go env file per network.

We also keep dry run config in a separate branch, which is why the
baklava config has been removed.

piersy committed:
Use pre-configured env files per network

As opposed to including instructions in the readme about how to modify
env files for each network.

Also introduces ENV_NAME to support multiple envs per network

We need this to be able to support the multiple baklava dry run
networks.

karlb commented on README.md:
```diff
@@ -80,33 +76,23 @@ git clone https://github.com/celo-org/simple-celo-node.git
 
 ### Optional configurations
 
+* **NODE_TYPE** - Choose the type of node you want to run:
+    * `full` (Full node) - A Full node contains a few recent blocks without historical states.
+    * `archive` (Archive node) - An Archive node stores the complete history of the blockchain, including historical states.
```
Do we support archive nodes? I didn't know that works with snap sync (which is the only sync mode supported according to the beginning of the README)

piersy replied:
Archive will work, enabling archive mode basically means we don't clear old state data, so it's just turning off the cleanup operation.

So with snap sync you will only have state from the block at which your snap sync completed.

karlb reviewed (commented):
I like having ready-made env files for each network, but do we really want to have the ENV_NAME abstraction on top of the NETWORK_NAME? It could cause more confusion than it solves. After all, there can only be one correct env for a network at a time. We can keep our dry-runs out of the main branch if we are afraid people might accidentally use their settings for normal operation.

piersy commented:
> I like having ready-made env files for each network, but do we really want to have the ENV_NAME abstraction on top of the NETWORK_NAME? It could cause more confusion than it solves. After all, there can only be one correct env for a network at a time. We can keep our dry-runs out of the main branch if we are afraid people might accidentally use their settings for normal operation.

I was thinking we would remove the dry runs and hence the extra abstraction once done, but I like the idea of keeping the dry runs in a separate branch. Will update to reflect this.

piersy committed:
Keep dry run config in separate branch

Moves the baklava dry run config out of the main branch
which removes the need for the ENV_NAME abstraction.
